backend/editor/api.py

```diff
@@ -421,7 +421,7 @@ async def create_entry_node(
     normalized_id = await taxonomy.create_entry_node(
         new_entry_node.name, new_entry_node.main_language_code
     )
-    await taxonomy.add_node_to_end(NodeType.ENTRY, normalized_id)
+    await taxonomy.add_node_to_end("ENTRY", normalized_id)
```

Comment:
you have to use .value
```suggestion
    await taxonomy.add_node_to_end(NodeType.ENTRY.value, normalized_id)
```

Reply:
I let you re-introduce the import @perierc

Reply:
BTW, to experiment: 
```python
docker compose run --rm taxonomy_api python3
>>> from editor.models.node_models import NodeType
>>> str(NodeType.ENTRY)
'NodeType.ENTRY'
>>> NodeType.ENTRY.value
'ENTRY'
```

Reply:
Thanks for this solution and for the tip to experiment!
Finally I found that using a `StrEnum` makes it work also, so I went with that